libs/watt/package/field/watt-field.component.ts

```diff
@@ -128,6 +129,7 @@ export class WattFieldComponent {
   errors = signal<ValidationErrors | null>(null);
   isRequired = signal(false);
   isEmpty = computed(() => this.errors()?.['required'] || this.errors()?.['rangeRequired']);
+  showErrors = input(true);
```

Comment:
if we can fix the validator to only run on `submit` as suggested above, then we don't need this input anymore